core/src/main/java/io/grpc/ServerBuilder.java

```diff
@@ -143,14 +143,21 @@ public T addStreamTracerFactory(ServerStreamTracer.Factory factory) {
    *
    * @param certChain file containing the full certificate chain
    * @param privateKey file containing the private key
-   *
    * @return this
    * @throws UnsupportedOperationException if the server does not support TLS.
    * @since 1.0.0
    */
   public abstract T useTransportSecurity(File certChain, File privateKey);
 
   /**
+   * Makes the server use CleartextUpgrade.
+   *
+   * @return this
+   * @throws UnsupportedOperationException if the server does not support CleartextUpgrade.
+   */
+  public abstract T usePlainTextUpgrade();
```

Comment:
Annotate with `@ExperimentalApi`. All new APIs start with the annotation.

Or, alternatively, just add the new method to `NettyServerBuilder`.